pyproject.toml

```diff
@@ -26,7 +26,7 @@ files = ["*/__init__.py"]
 folders = [{ path = "auth0" }]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.7"
```

Comment:
This matches 4.4.0 https://github.com/auth0/auth0-python/blob/4.4.0/setup.py#L34